R/dp_init.R

```diff
@@ -77,6 +77,25 @@ dp_init <- function(project_path = fs::path_wd(),
   commit_description <- "dp init"
   wd0 <- fs::path_wd()
 
+  if (!rlang::is_character(x = creds_set_dried)){
+    message("Encountered error in creds_set_dried in dp_init")
+    message("Make sure to use fn_dry in argument passed to creds_set_dried paramater.")
+    stop(cli::format_error(glue::glue("Do not supply the credentials directly as the function arguments.")))
+  }
+
+  if (!rlang::is_character(x = board_params_set_dried)){
+    message("Encountered error in board_params_set_dried in dp_init")
+    stop(cli::format_error(glue::glue("Make sure to use fn_dry in argument passed to board_params_set_dried paramater.")))
+  }
+
+  creds_set_dried_parsed <- rlang::parse_expr(creds_set_dried)
+
+  if (!class(creds_set_dried_parsed) == "call") {
+    message("Encountered error in creds_set_dried in dp_init")
+    message("Make sure to use fn_dry in argument passed to creds_set_dried paramater.")
+    stop(cli::format_error(glue::glue("Do not supply the credentials directly as the function arguments.")))
```

Comment:
For each if statement, recommend combining the message and stop calls into a single stop call for each, using glue for multi-line error messages.

Reply:
Done! Thanks.